scripts/do-gofmt.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+find_files() {
```

Comment:
let's remove the check-gofmt.sh if we are going to use this one

Reply:
Done